spec/features/toggle_language_spec.rb

```diff
@@ -0,0 +1,216 @@
+describe 'ToggleLanguage', js: true do
+  let(:user) { create :user1 }
+
+  after(:each) do
+    page.reset!
+  end
+
+  feature 'Toggling the locale dropdown to change the language' do
+    describe 'When signed out' do
```

Comment:
this is extremely minor, but I think all of these `describes` should be `contexts`, since you're describing similar functionality but in different contexts... (also they all start with "when," which is a good indicator that you're talking about a context)